hello_sdl_java/src/main/java/com/smartdevicelink/java/SdlService.java

```diff
@@ -405,4 +404,13 @@ public void onError(String error) {
             sdlManager.getScreenManager().presentChoiceSet(choiceSet, InteractionMode.MANUAL_ONLY);
         }
     }
+
+    private Vector<TTSChunk> createSimpleTTSChunks(String simple) {
+        if (simple == null) {
+            return null;
+        }
+        Vector<TTSChunk> chunks = new Vector<>();
+        chunks.add(new TTSChunk(simple, SpeechCapabilities.TEXT));
+        return chunks;
+    }
```

Comment:
Can we remove this method and replace the called to it with the porposed one liner 

`Collections.singletonList(new TTSChunk(simple, SpeechCapabilities.TEXT));`